splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/BroadcastJoinIT.java

```diff
@@ -34,7 +38,23 @@
  * @author Scott Fines
  *         Date: 5/20/15
  */
-public class BroadcastJoinIT{
+
+
+@RunWith(Parameterized.class)
+public class BroadcastJoinIT extends SpliceUnitTest {
+
+    private Boolean useSpark;
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        Collection<Object[]> params = Lists.newArrayListWithCapacity(4);
```

Comment:
size could be set to 2.

Reply:
OK, I will change it to 2.
Regarding the docs comment, I added a note for Gary in a JIRA comment on how to update broadcast join in the documentation.